share/completions/keybase.fish

```diff
@@ -9,123 +9,132 @@ function __fish_keybase_line_ends_with
-complete -c keybase -f -n "$ends keybase" -l pvl-kit
-complete -c keybase -f -n "$ends keybase" -l read-deleted-sigchain
-complete -c keybase -f -n "$ends keybase" -l remember-passphrase
-complete -c keybase -f -n "$ends keybase" -l run-mode
-complete -c keybase -f -n "$ends keybase" -l scraper-timeout
-complete -c keybase -f -n "$ends keybase" -l secret-keyring
-complete -c keybase -f -n "$ends keybase" -l server -s s
-complete -c keybase -f -n "$ends keybase" -l session-file
-complete -c keybase -f -n "$ends keybase" -l slow-gregor-conn
-complete -c keybase -f -n "$ends keybase" -l socket-file
-complete -c keybase -f -n "$ends keybase" -l standalone
-complete -c keybase -f -n "$ends keybase" -l timers
-complete -c keybase -f -n "$ends keybase" -l tor-hidden-address
-complete -c keybase -f -n "$ends keybase" -l tor-mode
-complete -c keybase -f -n "$ends keybase" -l tor-proxy
-complete -c keybase -f -n "$ends keybase" -l updater-config-file
-complete -c keybase -f -n "$ends keybase" -l use-default-log-file
-complete -c keybase -f -n "$ends keybase" -l use-root-config-file
-complete -c keybase -f -n "$ends keybase" -l user-cache-size
-complete -c keybase -f -n "$ends keybase" -l vdebug
-complete -c keybase -f -n "$ends keybase" -l version -s v
+# global options
+complete -c keybase -f -n "$ends keybase" -l api-dump-unsafe -d "Dump API call internals (may leak secrets)"
+complete -c keybase -f -n "$ends keybase" -l api-timeout -d "Set the HTTP timeout for API calls to the keybase API server"
+complete -c keybase -f -n "$ends keybase" -l api-uri-path-prefix -d "Specify an alternate API URI path prefix"
+complete -c keybase -f -n "$ends keybase" -l app-start-mode -d "Specify 'service' to auto-start UI app, or anything else to disable"
+complete -c keybase -f -n "$ends keybase" -l auto-fork -d "Enable auto-fork of background service"
+complete -c keybase -f -n "$ends keybase" -l bg-identifier-disabled -d "Supply to disable the BG identifier loop"
+complete -c keybase -f -n "$ends keybase" -l chat-db -d "Specify an alternate local Chat DB location"
+complete -c keybase -f -n "$ends keybase" -l code-signing-kids -d "Set of code signing key IDs (colon-separated)"
+complete -c keybase -f -n "$ends keybase" -l config-file -s c -d "Specify an (alternate) master config file"
+complete -c keybase -f -n "$ends keybase" -l db -d "Specify an alternate local DB location"
+complete -c keybase -f -n "$ends keybase" -l debug -s d -d "Enable debugging mode"
+complete -c keybase -f -n "$ends keybase" -l debug-journeycard -d "Enable experimental journey cards"
+complete -c keybase -f -n "$ends keybase" -l disable-bg-conv-loader -d "Disable background conversation loading"
+complete -c keybase -f -n "$ends keybase" -l disable-cert-pinning -d "Disable certificate pinning within the app. WARNING: This reduces the security of the app. Do not use unless necessary! This should only be used if you are running keybase behind a proxy that does TLS interception"
```

Comment:
can you shorten this and similar ones (maybe truncate after the first sentence).
We try to keep it around 60 columns
Could maybe replace it with `(insecure!)` or similar.